src/lib/components/Widget.js

```diff
@@ -102,7 +102,7 @@ export const Widget = React.forwardRef((props, forwardedRef) => {
       setSrc(src);
     } else if (srcOrCode?.code) {
       setCode(srcOrCode.code);
-      setSrc(null);
+      setSrc(near.features.enableWidgetSrcWithCodeOverride ? propsSrc : null);
```

Comment:
This may throw undefined error

Reply:
I believe `near.features` will always be an object thanks to the default here in `near.js`:

<img width="360" alt="Screen Shot 2024-01-03 at 11 01 50 AM" src="https://github.com/NearSocial/VM/assets/1475067/bfaf2850-378a-4332-8aff-d0a6f312f0d9">

Reply:
I'll open up a PR to convert to `near?.features?.enableWidgetSrcWithCodeOverride` just to be extra safe.

Reply:
@evgenykuzyakov https://github.com/NearSocial/VM/pull/163